plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -439,6 +439,8 @@ def deserialize(self, serialized):
         meta_data["base_file"] = self.id
         meta_data["status"] = "unknown"  # TODO: Add material verfication
 
+        global_setting_values = {}
```

Comment:
Note that I moved this line of code from line 489, I did not introduce this variable name.